services/wallet/service.go

```diff
@@ -61,8 +62,8 @@ func NewService(
 	feed *event.Feed,
 	mediaServer *server.MediaServer,
 ) *Service {
-	cryptoOnRampManager := NewCryptoOnRampManager(&CryptoOnRampOptions{
-		dataSourceType: DataSourceStatic,
+	cryptoOnRampManager := onramp.NewManager(&onramp.Options{
```

Comment:
it seems like we are setting default to Static? is this a todo to also support fetch from http? 
I am not sure I understand when we would do a static fetch and when we would perform a http request in case of on ramp providers?

Reply:
So... This code was done like 3 years ago and apparently the original spec was to be able to fetch the list of providers from the web (so we don't need to release a new Status version to change it). However, there were some issues with that approach, a static list was introduced and the http source was left as eternal tech debt :-)
So, this whole thing could use some cleanup/rework, but there's no real need for that at the moment. I just moved some code around for better organization, but tried changing as little as possible.

Reply:
Also, the diff is messed up, there's almost no new code other than:
- Moving the list of providers to a separate file
- Specifying the static list of providers as structs instead of JSON strings
- Adding chains and recurrentURL fields to struct